Update Kubeapps on TKG tutorial to comply style guidelines (#5298)
vmware-tanzu/kubeapps: site/content/docs/latest/tutorials/kubeapps-on-tkg/step-2.md

   - Replace the `OIDC-REDEEM-URL` with the token redeem URL of the OIDC provider. For CSP it is `https://console.cloud.vmware.com/csp/gateway/am/api/auth/token`.
   - Replace the `OIDC-JWKS-URL` with the JSON Web Key Set URL of the OIDC provider. For CSP it is `https://console.cloud.vmware.com/csp/gateway/am/api/auth/token-public-key?format=jwks`.

> > **TIP**: Remember that any OIDC-compliant provider should expose a `.well-known/openid-configuration` ([CSP example](https://console.cloud.vmware.com/csp/gateway/am/api/.well-known/openid-configuration)) where you are able to find the required endpoints in this step.

Reviewer: ```suggestion
> **TIP**: Remember that any OIDC-compliant provider should expose a `.well-known/openid-configuration` ([CSP example](https://console.cloud.vmware.com/csp/gateway/am/api/.well-known/openid-configuration)) where you can find the required endpoints in this step.
```

   kubectl port-forward -n kubeapps svc/kubeapps 8080:80
   ```

>    This starts an HTTP proxy for secure access to the Kubeapps dashboard.

Reviewer: ```suggestion
   This command spawns an HTTP proxy providing secure access to the Kubeapps dashboard.
```

Author: Thanks for the revision. Updated!
